src/app/pages/RewardPage/components/RewardsDetail/index.tsx

```diff
@@ -111,7 +120,16 @@ export const RewardsDetail: React.FC<IRewardsDetailProps> = ({
         <LoadableValue
           value={
             bignumber(totalEarnedAmount).greaterThan(0) ? (
-              <Tooltip content={`${totalEarnedAmount} ${asset}`}>
+              <Tooltip
```

Comment:
You don't need this `Tooltip`, just use `useTooltip` in `AssetValue` component and it will be handled automatically.